[2024-12-06T22:08:20.320+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2024-12-06T22:08:11.810815+00:00 [queued]>
[2024-12-06T22:08:20.331+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2024-12-06T22:08:11.810815+00:00 [queued]>
[2024-12-06T22:08:20.332+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-06T22:08:20.341+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_load_weather_data> on 2024-12-06 22:08:11.810815+00:00
[2024-12-06T22:08:20.348+0000] {standard_task_runner.py:60} INFO - Started process 8920 to run task
[2024-12-06T22:08:20.352+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data', 'manual__2024-12-06T22:08:11.810815+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmpr6u39cz8']
[2024-12-06T22:08:20.353+0000] {standard_task_runner.py:88} INFO - Job 88: Subtask transform_load_weather_data
[2024-12-06T22:08:20.368+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-06T22:08:20.407+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data manual__2024-12-06T22:08:11.810815+00:00 [running]> on host 8a5e48d7ee4d
[2024-12-06T22:08:20.494+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T22:08:11.810815+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T22:08:11.810815+00:00'
[2024-12-06T22:08:20.511+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-06T22:08:20.522+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data, execution_date=20241206T220811, start_date=20241206T220820, end_date=20241206T220820
[2024-12-06T22:08:20.563+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-06T22:08:20.593+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
